src/chrome/content/rules/baden-wuerttemberg.de.xml

```diff
@@ -143,10 +146,6 @@
 	<target host="www.vm.baden-wuerttemberg.de" />
 	<target host="wm.baden-wuerttemberg.de" />
 
-	<test url="http://www.beteiligungsportal.baden-wuerttemberg.de/" />
-	<test url="http://www.mvi.baden-wuerttemberg.de/" />
-	<test url="http://www.mwk.baden-wuerttemberg.de/" />
```

Comment:
They need to be added as targets, see rules below.

Reply:
Sorry, my wrong.